modules/nodes-nodes-jobs-about.adoc

```diff
@@ -0,0 +1,109 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-pods-jobs.adoc
+
+[id='nodes-pods-configuring-replication_{context}']
+= About jobs 
+
+A job tracks the overall progress of a task and updates its status with information
+about active, succeeded, and failed pods. Deleting a job will clean up any pod
+replicas it created. Jobs are part of the Kubernetes API, which can be managed
```

Comment:
I would not use word replicas here, it might be confusing. Just say any pod (or any number of pods) it created.